### PR TITLE
docs(CONTRIBUTING): grammar updates

### PR DESCRIPTION
**Pull request by gargarchit**
Signed-off-by: Archit garg4.ag@gmail.com

I found some Improper word choice as well as Improper formatting for that I have made changes and corrected that so, now It will be easier for everyone to read.
Please Merge this

### CODE REVIEW

**Comment by gargarchit** [comment]
Thank You @coveralls

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -96,23 +96,23 @@ Before you submit your pull request consider the following guidelines:
...
 
-* Before creating the Pull Request, ensure your branch sits on top of master (as opposed to branch off a branch). This ensures the reviewer will need only minimal effort to integrate your work by fast-fowarding master:
+* Before creating the Pull Request, ensure your branch sits on top of master (as opposed to branch off a branch). This ensures the reviewer will need only minimal effort to integrate your work by the fast-forwarding master:
 
   ```text
     git rebase upstream/master
   ```
 
-* Last step before creating the Pull Request, package and run all tests a last time:
+* Last step before creating the Pull Request, package and run all tests the last time:
```
Comment (irmerk) [inline CONTRIBUTING.md]: This edit is unnecessary.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -96,23 +96,23 @@ Before you submit your pull request consider the following guidelines:
...
 
   ```text
     git commit -a
   ```
 
-  Note: the optional commit `-a` command line option will automatically "add" and "rm" edited files.
+  Note: the optional commit `-a` command-line option will automatically "add" and "rm" edited files.
 
-* Before creating the Pull Request, ensure your branch sits on top of master (as opposed to branch off a branch). This ensures the reviewer will need only minimal effort to integrate your work by fast-fowarding master:
+* Before creating the Pull Request, ensure your branch sits on top of master (as opposed to branch off a branch). This ensures the reviewer will need only minimal effort to integrate your work by the fast-forwarding master:
```
Comment (irmerk) [inline CONTRIBUTING.md]: "the" in this edit is unnecessary.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -96,23 +96,23 @@ Before you submit your pull request consider the following guidelines:
...
 
-* Commit your changes using a descriptive commit message that follows our [commit message conventions][developers.commits]. Adherence to the [commit message conventions][developers.commits] is required, because release notes are automatically generated from these messages.
+* Commit your changes using a descriptive commit message that follows our [commit message conventions][developers.commits]. Adherence to the [commit message conventions][developers.commits] is required because release notes are automatically generated from these messages.
 
   ```text
     git commit -a
   ```
 
-  Note: the optional commit `-a` command line option will automatically "add" and "rm" edited files.
+  Note: the optional commit `-a` command-line option will automatically "add" and "rm" edited files.
```
Comment (irmerk) [inline CONTRIBUTING.md]: This edit is unnecessary.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ Do not open issues for general support questions as we want to keep GitHub issue
 Stack Overflow is a much better place to ask questions since:
 
 * there are thousands of people willing to help on Stack Overflow
-* questions and answers stay available for public viewing so your question / answer might help someone else
+* Questions and answers stay available for public viewing so your question/answer might help someone else
```
Comment (irmerk) [inline CONTRIBUTING.md]: This edit is unnecessary.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -67,9 +67,9 @@ For large fixes, please build and test the documentation before submitting the P
 
 Before you submit your issue search the archive, maybe your question was already answered.
 
-If your issue appears to be a bug, and hasn't been reported, open a new issue. Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues.
+If your issue appears to be a bug and hasn't been reported, open a new issue. Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues.
 
-The "[new issue][github-new-issue]" form contains a number of prompts that you should fill out to make it easier to understand and categorize the issue.
+The "[new issue][github-new-issue]" form contains several prompts that you should fill out to make it easier to understand and categorize the issue.
```
Comment (irmerk) [inline CONTRIBUTING.md]: This edit is unnecessary.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -96,23 +96,23 @@ Before you submit your pull request consider the following guidelines:
 * Follow our [Coding Rules][developers.rules].
 * Ensure you provide a DCO sign-off for your commits using the -s option of git commit. For more information see [how this works][dcohow].
 * If the changes affect public APIs, change or add relevant [documentation][developers.documentation].
-* Run the [unit][developers.unit-tests] test suite, and ensure that all tests pass.
+* Run the [unit][developers.unit-tests] the test suite, and ensure that all tests pass.
 
-* Commit your changes using a descriptive commit message that follows our [commit message conventions][developers.commits]. Adherence to the [commit message conventions][developers.commits] is required, because release notes are automatically generated from these messages.
+* Commit your changes using a descriptive commit message that follows our [commit message conventions][developers.commits]. Adherence to the [commit message conventions][developers.commits] is required because release notes are automatically generated from these messages.
```
Comment (irmerk) [inline CONTRIBUTING.md]: This edit is unnecessary.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -226,4 +226,4 @@ Accord Project documentation files are made available under the [Creative Common
 [clacorp]: http://code.google.com/legal/corporate-cla-v1.0.html
 
 [apache]: https://github.com/accordproject/ergo/blob/master/LICENSE
-[creativecommons]: http://creativecommons.org/licenses/by/4.0/
\ No newline at end of file
+[creativecommons]: http://creativecommons.org/licenses/by/4.0/
```
Comment (irmerk) [inline CONTRIBUTING.md]: This edit is unnecessary.

File: CONTRIBUTING.md (hunk 1 above)
Comment (gargarchit) [reply CONTRIBUTING.md]: Updated

File: CONTRIBUTING.md (hunk 2 above)
Comment (gargarchit) [reply CONTRIBUTING.md]: Updated

File: CONTRIBUTING.md (hunk 3 above)
Comment (gargarchit) [reply CONTRIBUTING.md]: Updated

File: CONTRIBUTING.md (hunk 4 above)
Comment (gargarchit) [reply CONTRIBUTING.md]: Updated

File: CONTRIBUTING.md (hunk 5 above)
Comment (gargarchit) [reply CONTRIBUTING.md]: Updated

File: CONTRIBUTING.md (hunk 6 above)
Comment (gargarchit) [reply CONTRIBUTING.md]: Updated

File: CONTRIBUTING.md (hunk 7 above)
Comment (gargarchit) [reply CONTRIBUTING.md]: There is no change in this

**Comment by gargarchit** [comment]
Thank You @irmerk 
I added Add a DCO (Developers Certificate of Originality)

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -96,7 +96,7 @@ Before you submit your pull request consider the following guidelines:
 * Follow our [Coding Rules][developers.rules].
 * Ensure you provide a DCO sign-off for your commits using the -s option of git commit. For more information see [how this works][dcohow].
 * If the changes affect public APIs, change or add relevant [documentation][developers.documentation].
-* Run the [unit][developers.unit-tests] test suite, and ensure that all tests pass.
+* Run the [unit][developers.unit-tests] the test suite, and ensure that all tests pass.
```
Comment (irmerk) [inline CONTRIBUTING.md]: This still needs to be removed.

File: CONTRIBUTING.md (hunk 7 above)
Comment (irmerk) [inline CONTRIBUTING.md]: This still needs to be removed.

File: CONTRIBUTING.md (hunk 8 above)
Comment (gargarchit) [reply CONTRIBUTING.md]: Updated

File: CONTRIBUTING.md (hunk 7 above)
Comment (gargarchit) [reply CONTRIBUTING.md]: Hi @irmerk I didn't make any changes in this but it is showing it has changed in all changes but not reflecting any changes

**Comment by gargarchit** [comment]
Thank You @irmerk for approving and merging my pull request

**Pull request merged**